app/controllers/admin/group_roles_controller.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Admin
+  # OVERRIDE from AdminController inheretence for user roles authorization
+  class GroupRolesController < ApplicationController
+    before_action :load_group
+    before_action :cannot_remove_admin_role_from_admin_group, only: [:remove]
+    layout 'hyrax/dashboard'
+
+    rescue_from ActiveRecord::RecordNotFound, with: :redirect_not_found
+
+    def index
+      # OVERRIDE: AUTHORIZE AN EDIT ROLE TO ACCESS THE ROLES INDEX
+      authorize! :edit, Hyrax::Group
+      add_breadcrumb t(:'hyrax.controls.home'), root_path
+      add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
+      add_breadcrumb t(:'hyku.admin.groups.title.edit'), edit_admin_group_path(@group)
+      add_breadcrumb t(:'hyku.admin.groups.title.roles'), request.path
+
+      @roles = ::Role.site - @group.roles
+      render template: 'admin/groups/roles'
+    end
+
+    def add
```

Comment:
I hate to be "that guy" but why aren't these named create and destroy?

Reply:
The buttons in the UI say "add" and "remove", and I think at the time I was thinking in terms of "adding/removing a role to/from a group", not "creating/deleting a GroupRole". But fast forward to present day and I agree that "self" in the `GroupRolesController` should refer to the `GroupRole` records, so I will update these

Reply:
@orangewolf actually, it looks like I was following the pattern set by this controller's sister, [GroupUsersController](https://github.com/samvera/hyku/blob/f077f4603ac2d1edc3f7ef5f9b10524cb57f08a8/app/controllers/admin/group_users_controller.rb#L16-L28). Do you want me to change them both to use "create/destroy", or keep these at "add/remove"?